main.go

```diff
@@ -44,6 +48,27 @@ func createApp() *cli.App {
 	return app
 }
 
+func addBefore(app *cli.App, debug *bool) {
```

Comment:
remove `debug *bool` as not needed anymore, see comment below

Reply:
Jep.